src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] infoFromFile = readFromFile(fromFileName);
+        writeToFile(toFileName, createReport(infoFromFile));
+    }
+
+    private String[] readFromFile(String fromFileName) {
+        File file = new File(fromFileName);
+        try (BufferedReader reader = new BufferedReader(new FileReader(file))) {
+            StringBuilder builder = new StringBuilder();
+            String value = reader.readLine();
+            while (value != null) {
+                builder.append(value).append(LINE_SEPARATOR);
+                value = reader.readLine();
+            }
+            return builder.toString().split(LINE_SEPARATOR);
```

Comment:
Why do you need to build and after split data? Can you simplify it?

Reply:
I've been thinking and haven't found the best option. can you give me a hint?